tribits/examples/TribitsExampleProject/README.md

```diff
@@ -19,16 +20,28 @@ with:
     -DTribitsExProj_TRIBITS_DIR=<tribits-dir> \
     -DTribitsExProj_ENABLE_TESTS=ON \
     -DTribitsExProj_ENABLE_ALL_PACKAGES=ON
+    -DTribitsExProj_ENABLE_SECONDARY_TESTED_CODE=ON \
     -DCMAKE_CXX_COMPILER=g++ \
     <path-to-TribitsExampleProject>
 ```
 
-and then build and test with:
+then build and test with:
 
 ```
   make -j4 && ctest -j4
 ```
 
+and then install:
+
+```
+  make DESTDIR=<path-to-install-dir> install
+```
+
+`TribitsExampleProject` will be installed to 
+`<path-to-install-dir>/usr/local/` and this path should be then used 
```

Comment:
Is that true?  It will install to `<path-to-install-dir>/usr/local/`?  Does `DESTDIR` not just override the configure-time option `CMAKE_INSTALL_PREFIX`?

Reply:
Hello Ross,
please find logs below:
```shell
 marcin@pop-os ~/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/build 
 ls -la ../install  
total 8
drwxrwxr-x 2 marcin marcin 4096 lut  4 10:47 .
drwxrwxr-x 7 marcin marcin 4096 lut  4 10:47 ..

 marcin@pop-os ~/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/build 
 make DESTDIR=/home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install install
Consolidate compiler generated dependencies of target simplecxx
[  5%] Built target simplecxx
Consolidate compiler generated dependencies of target simplecxx-helloworld
[ 10%] Built target simplecxx-helloworld
Consolidate compiler generated dependencies of target SimpleCxx_HelloWorldTests
[ 16%] Built target SimpleCxx_HelloWorldTests
Consolidate compiler generated dependencies of target mixedlang
[ 29%] Built target mixedlang
Consolidate compiler generated dependencies of target MixedLang_RayTracerTests
[ 35%] Built target MixedLang_RayTracerTests
Consolidate compiler generated dependencies of target pws_a
[ 40%] Built target pws_a
Consolidate compiler generated dependencies of target WithSubpackagesA_a_test
[ 45%] Built target WithSubpackagesA_a_test
Consolidate compiler generated dependencies of target pws_b
[ 51%] Built target pws_b
Consolidate compiler generated dependencies of target WithSubpackagesB_b_test
[ 56%] Built target WithSubpackagesB_b_test
Consolidate compiler generated dependencies of target b_test_utils
[ 62%] Built target b_test_utils
Consolidate compiler generated dependencies of target b_mixed_lang
[ 67%] Built target b_mixed_lang
Consolidate compiler generated dependencies of target WithSubpackagesB_test_of_b_mixed_lang
[ 72%] Built target WithSubpackagesB_test_of_b_mixed_lang
Consolidate compiler generated dependencies of target c_util
[ 78%] Built target c_util
Consolidate compiler generated dependencies of target pws_c
[ 83%] Built target pws_c
Consolidate compiler generated dependencies of target WithSubpackagesC_c_test
[ 89%] Built target WithSubpackagesC_c_test
Consolidate compiler generated dependencies of target c_b_mixed_lang
[ 94%] Built target c_b_mixed_lang
Consolidate compiler generated dependencies of target WithSubpackagesC_test_of_c_b_mixed_lang
[100%] Built target WithSubpackagesC_test_of_c_b_mixed_lang
Install the project...
-- Install configuration: "RELEASE"
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/external_packages/HeaderOnlyTpl/HeaderOnlyTplConfig.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/external_packages/HeaderOnlyTpl/HeaderOnlyTplConfigVersion.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/include/TribitsExProj_version.h
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/libsimplecxx.a
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/include/SimpleCxx_config.h
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/include/SimpleCxx_HelloWorld.hpp
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/bin/simplecxx-helloworld
-- Set runtime path of "/home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/bin/simplecxx-helloworld" to "/usr/local/lib"
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/SimpleCxx/SimpleCxxConfig.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/SimpleCxx/SimpleCxxTargets.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/SimpleCxx/SimpleCxxTargets-release.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/libmixedlang.a
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/include/MixedLang_config.h
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/include/MixedLang.hpp
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/include/Ray_Tracer.hh
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/include/Ray.hh
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/MixedLang/MixedLangConfig.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/MixedLang/MixedLangTargets.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/MixedLang/MixedLangTargets-release.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/libpws_a.a
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/include/A.hpp
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/WithSubpackagesA/WithSubpackagesAConfig.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/WithSubpackagesA/WithSubpackagesATargets.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/WithSubpackagesA/WithSubpackagesATargets-release.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/libpws_b.a
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/include/B.hpp
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/include/WithSubpackagesB_config.h
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/share/WithSubpackagesB/stuff
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/share/WithSubpackagesB/stuff/regular_file.txt
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/share/WithSubpackagesB/stuff/exec_script.sh
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/WithSubpackagesB/WithSubpackagesBConfig.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/WithSubpackagesB/WithSubpackagesBTargets.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/WithSubpackagesB/WithSubpackagesBTargets-release.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/libpws_c.a
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/include/wsp_c/C.hpp
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/WithSubpackagesC/WithSubpackagesCConfig.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/WithSubpackagesC/WithSubpackagesCTargets.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/WithSubpackagesC/WithSubpackagesCTargets-release.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/WithSubpackages/WithSubpackagesConfig.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/WithSubpackages/WithSubpackagesTargets.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/TribitsExProj/TribitsExProjConfig.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/lib/cmake/TribitsExProj/TribitsExProjConfigVersion.cmake
-- Installing: /home/marcin/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/install/usr/local/include/TribitsExProjConfig.cmake

 marcin@pop-os ~/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/build 
 ls -la ../install                                                                                                           
total 12
drwxrwxr-x 3 marcin marcin 4096 lut  4 10:50 .
drwxrwxr-x 7 marcin marcin 4096 lut  4 10:47 ..
drwxrwxr-x 3 marcin marcin 4096 lut  4 10:50 usr

 marcin@pop-os ~/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/build 
 ls -la ../install/usr
total 12
drwxrwxr-x 3 marcin marcin 4096 lut  4 10:50 .
drwxrwxr-x 3 marcin marcin 4096 lut  4 10:50 ..
drwxrwxr-x 6 marcin marcin 4096 lut  4 10:50 local

 marcin@pop-os ~/Dokumenty/NGA/projects/tribits_test/TriBITS/tribits/examples/TribitsExampleProject/build 
 ls -la ../install/usr/local 
total 24
drwxrwxr-x 6 marcin marcin 4096 lut  4 10:50 .
drwxrwxr-x 3 marcin marcin 4096 lut  4 10:50 ..
drwxrwxr-x 2 marcin marcin 4096 lut  4 10:50 bin
drwxrwxr-x 3 marcin marcin 4096 lut  4 10:50 include
drwxrwxr-x 4 marcin marcin 4096 lut  4 10:50 lib
drwxrwxr-x 3 marcin marcin 4096 lut  4 10:50 share
```